sitecore-xp0-sxa-horizon/docker/docker-compose.override.yml

```diff
@@ -0,0 +1,148 @@
+      - ${LICENSE_PATH}:C:\license
+    entrypoint: powershell -Command "& C:\tools\entrypoints\iis\Development.ps1"
+    mem_limit: ${MEM_LIMIT_CM:-4GB}
+    restart: always
+    labels:
+      - "traefik.http.middlewares.redirect-to-https.redirectScheme.scheme=https"
+      - "traefik.http.routers.cm.entrypoints=web"
+      - "traefik.http.routers.cm.rule=Host(`${CM_HOST}`)"
+      - "traefik.http.routers.cm.middlewares=redirect-to-https"
+
+  id:
+    environment:
+      Sitecore_Sitecore__IdentityServer__Clients__DefaultClient__AllowedCorsOrigins__AllowedCorsOriginsGroup2: https://${HRZ_HOST}
+      SITECORE_LICENSE_LOCATION: C:\license\license.xml
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  xconnect:
+    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-xconnect:${VERSION:-latest}
+    build:
+      context: ./build/xconnect
+      args:
+        BASE_IMAGE: ${SITECORE_DOCKER_REGISTRY}sitecore-xp0-xconnect:${SITECORE_VERSION}
+        TOOLING_IMAGE: ${SITECORE_TOOLS_REGISTRY}sitecore-docker-tools-assets:${TOOLS_VERSION}
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - .\data\xconnect:C:\deploy
+      - ${LICENSE_PATH}:C:\license
+    entrypoint: powershell -Command "& C:\tools\entrypoints\iis\Development.ps1"
+
+  xdbsearchworker:
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  xdbautomationworker:
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  cortexprocessingworker:
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  horizon:
+    image: scr.sitecore.com/sxp/modules/sitecore-horizon:${HRZ_VERSION}
+    isolation: ${ISOLATION}
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost/healthz/live"]
+      timeout: 300s
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license\license.xml
+      Sitecore_License: ${SITECORE_LICENSE}
+      Sitecore_FederatedUI__HostBaseUrl: http://horizon
+      Sitecore_SitecorePlatform__ContentManagementUrl: https://${CM_HOST}
+      Sitecore_SitecorePlatform__ContentManagementInternalUrl: http://cm
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__RequireHttpsMetadata: "false"
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__Authority: https://${ID_HOST}
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__CallbackAuthority: https://${HRZ_HOST}
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__InternalAuthority: http://id
+      Sitecore_Sitecore__Authentication__BearerAuthenticationOptions__Authority: https://${ID_HOST}
+      Sitecore_Sitecore__Authentication__BearerAuthenticationOptions__InternalAuthority: http://id
+      Sitecore_Sitecore__Authentication__BearerAuthenticationOptions__RequireHttpsMetadata: "false"
```

Comment:
Need to add `Sitecore_Plugins__Filters__ExperienceAccelerator: +SXA` environment variable here for this SXA build

Reply:
Why this here. It runs without it?